modules/standard/AutoMath.chpl

```diff
@@ -174,9 +177,18 @@ module AutoMath {
       return cabs(z);
   }
 
-
-  /* Returns the real phase angle of complex argument `z`. */
+  // When removing this deprecated function, be sure to remove chpl_carg and
+  // move its contents into Math.chpl to reduce the symbols living in this
+  // module.
+  pragma "last resort"
+  @chpldoc.nodoc
+  @deprecated(notes="carg will soon stop being included by default, please 'use' or 'import' the 'Math' module to call it")
```

Comment:
"soon stop" sounds kinda odd to me. Maybe: "In an upcoming release 'carg' will no longer be included by default, ..."